### ISSUE TITLE
[Chips] Update chips readme with accessibility recommendations

### ISSUE THREAD

**Issue by jmdetloff** [material-components/material-components-ios#4316]
---
#### Internal data
- Associated internal bug: [b/117179000](http://b/117179000)

**Comment by jverkoey** [comment]
@jmdetloff What are the a11y recommendations we should be documenting in this component?

**Comment by romoore** [comment]
@jmdetloff Friendly ping to briefly list what needs to be documented.

**Comment by material-mirror** [comment]
The internal issue [b/117179000](http://b/117179000) is now closed. This issue is being closed as a result.
